docs/aws_cli_config.md

```diff
@@ -0,0 +1,159 @@
+# AWS CLI Configuration Guide
+
+To configure the AWS CLI, we recommend you start with this AWS tutorial on getting started
+with the CLI: [AWS CLI Tutorial](http://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-started.html).
+
+## Configuring AWS Credentials
+
+To use API keys for automated access to AWS, you must properly configure the `aws` utility.
+
+Start by running `aws configure`, which will configure your AWS access credentials. These
+keys should correspond to your main (top-level) account, or your principal service account
+within the main account.
+
+```
+$ aws configure
+AWS Access Key ID [None]: AKIAIOSFODNN7EXAMPLE
+AWS Secret Access Key [None]: wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY
+Default region name [None]: us-east-1
+Default output format [None]:
+```
+
+This will create a config file `~/.aws/config` with configuration variable values for your AWS account, 
+and a credentials file `~/.aws/credentials` with API keys for your account (this is sensitive information).
+
+An example `~/.aws/config` file:
+
+```
+[profile my-red-profile]
+role_arn = arn:aws:iam::123456789:role/red-developer
+role_session_name = whoami@ucsc.edu
+source_profile = red-creds
+```
+
+An example `~/.aws/credentials` file:
+
+```
+[red-creds]
+aws_access_key_id = AKIAIOSFODNN7EXAMPLE
+aws_secret_access_key = wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY 
+```
+
+Note that `source_profile` in the config file refers to the label given to the set of credentials
+in the credentials file.
+
+To test out the profile you just created, call the `aws` cli utility and specify the profile:
+
+```
+aws s3 ls --profile my-red-profile
+```
+
+## Enabling Multi-Factor Authentication
+
+If you have multi-factor authentication enabled, you should specify your MFA device
+in your AWS config file using the `mfa_serial` variable. Start by logging into the
+AWS console, and finding the name of your MFA device. It will use the ARN number for
+your top-level organization, and is typically something like
+
+```
+mfa_serial = arn:aws:iam::123456789:mfa/whoami@ucsc.edu
+```
+
+Once you have added this MFA device and enabled MFA, test it by running a simple
+list-buckets action. You should be propmted for your 2FA code:
+
+```
+$ aws s3 ls
+Enter MFA code for arn:aws:iam::652235167018:mfa/chmreid@ucsc.edu:
```

Comment:
maybe drop the account number?  unless its already in the env file.

Reply:
thanks for catching that, fixed in ba6bf71